content/_data/taxonomy.yaml

```diff
@@ -1,10 +1,4 @@
 post:
-  - tag: Talk
-    icon: talk
-    img: true
-  - tag: Workshop
-    icon: workshop
-    img: true
```

Comment:
I don't fully understand the logic, but these types of posts still appear in lists on e.g. oss-detail pages (https://deploy-preview-290--oddleventy.netlify.app/accoutrement/). But now they're missing little corner icons, and they link to the /tags/ page.

Reply:
Like we talked about in the meeting this morning, I think it would be fine (even useful?) for talks/workshops to show up as related posts for Birds and OSS projects and in tag lists. Looks like they lost their images and icons, though.

Reply:
Should be fixed now